src/middleware/github-webhook-middleware.ts

```diff
@@ -99,14 +99,19 @@ export const GithubWebhookMiddleware = (
 		const orgName = payload?.repository?.owner?.login || "none";
 		const gitHubInstallationId = Number(payload?.installation?.id);
 
+		const subscriptions = await Subscription.getAllForInstallation(gitHubInstallationId);
+		const jiraHost = subscriptions.length ? subscriptions[0].jiraHost : undefined;
 		context.log = getLogger("github.webhooks", {
-			webhookId,
-			gitHubInstallationId,
-			event: webhookEvent,
-			webhookReceived,
-			repoName,
-			orgName,
-			...cloneAllowedLogFields(context.log.fields)
+			level: await stringFlag(StringFlags.LOG_LEVEL, defaultLogLevel, jiraHost),
+			fields: {
+				webhookId,
+				gitHubInstallationId,
+				event: webhookEvent,
+				webhookReceived,
+				repoName,
+				orgName,
+				...cloneAllowedLogFields(context.log.fields)
```

Comment:
cloneAllowedLogFields probably isn't required anymore, since `name` is a top level restricted attribute and we are inside fields..... however name and then field.name both existing could be confusing.

Reply:
I'll leave it as is for now :)

Reply:
okay, removed it.